dnf/yum/config.py

```diff
@@ -738,7 +738,7 @@ def releasever(self, val):
 
     keepcache = BoolOption(False)
     logdir = Option('/var/log') # :api
-    reposdir = ListOption(['/etc/yum/repos.d', '/etc/yum.repos.d']) # :api
+    reposdir = ListOption(['/etc/yum.repos.d', '/etc/yum/repos.d'])  # :api
```

Comment:
This is being separately addressed in #416. Would the fix there work for this, too?